OtripleS.Web.Api.Tests.Acceptance/APIs/StudentExams/StudentExamsApiTests.cs

```diff
@@ -0,0 +1,255 @@
+            DateTimeOffset now = DateTimeOffset.UtcNow;
+            Guid userId = Guid.NewGuid();
+
+            var filler = new Filler<Exam>();
+
+            filler.Setup()
+                .OnProperty(exam => exam.CreatedBy).Use(userId)
+                .OnProperty(exam => exam.UpdatedBy).Use(userId)
+                .OnProperty(exam => exam.CreatedDate).Use(now)
+                .OnProperty(exam => exam.UpdatedDate).Use(now)
+                .OnProperty(exam => exam.SemesterCourseId).Use(semesterCourse.Id)
+                .OnType<DateTimeOffset>().Use(GetRandomDateTime());
+
+            return filler;
+        }
+
+        private async ValueTask<SemesterCourse> PostRandomSemesterCourse(
+            Teacher teacher,
+            Course course,
+            Classroom classroom)
+        {
+            SemesterCourse semesterCourse =
+                CreateRandomSemesterCourseFiller(teacher, course, classroom).Create();
+
+            return await this.otripleSApiBroker.PostSemesterCourseAsync(semesterCourse);
+        }
+
+        private Filler<SemesterCourse> CreateRandomSemesterCourseFiller(
+            Teacher teacher,
+            Course course,
+            Classroom classroom)
+        {
+            DateTimeOffset now = DateTimeOffset.UtcNow;
+            Guid userId = Guid.NewGuid();
+
+            var filler = new Filler<SemesterCourse>();
+
+            filler.Setup()
+                .OnProperty(semesterCourse => semesterCourse.CreatedBy).Use(userId)
+                .OnProperty(semesterCourse => semesterCourse.UpdatedBy).Use(userId)
+                .OnProperty(semesterCourse => semesterCourse.CreatedDate).Use(now)
+                .OnProperty(semesterCourse => semesterCourse.UpdatedDate).Use(now)
+                .OnProperty(semesterCourse => semesterCourse.TeacherId).Use(teacher.Id)
+                .OnProperty(semesterCourse => semesterCourse.CourseId).Use(course.Id)
+                .OnProperty(semesterCourse => semesterCourse.ClassroomId).Use(classroom.Id)
+                .OnType<DateTimeOffset>().Use(GetRandomDateTime());
+
+            return filler;
+        }
+
+        private async ValueTask<Course> PostRandomCourse()
+        {
+            Course course = CreateRandomCourseFiller().Create();
+
+            return await this.otripleSApiBroker.PostCourseAsync(course);
+        }
+
+        private Filler<Course> CreateRandomCourseFiller()
+        {
+            DateTimeOffset now = DateTimeOffset.UtcNow;
+            Guid userId = Guid.NewGuid();
+
+            var filler = new Filler<Course>();
+
+            filler.Setup()
+                .OnProperty(course => course.CreatedBy).Use(userId)
+                .OnProperty(course => course.UpdatedBy).Use(userId)
+                .OnProperty(course => course.CreatedDate).Use(now)
+                .OnProperty(course => course.UpdatedDate).Use(now)
+                .OnType<DateTimeOffset>().Use(GetRandomDateTime());
+
+            return filler;
+        }
```

Comment:
fillers go to the bottom - the Standard is that methods that are being used within the same file, stay at the bottom - but the methods that are being used outside of the file should go to the top.